src/cameras/hooks/use-get-cameras.js

```diff
@@ -0,0 +1,13 @@
+import { useEffect } from 'react'
+import { useDispatch } from 'react-redux'
+
+import { actions } from 'cameras/slices/get'
+
+export const useGetCameras = () => {
```

Comment:
Since this hook is not part of the present task, let's remove it and re-implement it when the time comes.

Reply:
done